WordPress/pattern-directory
# Theme: Add og and twitter meta tags for better search and sharing

Adds `og:*` and `twitter:*` tags for the homepage, category archives, and single pattern pages.

### Screenshots

<img width="916" alt="Screen Shot 2021-07-20 at 11 47 41 AM" src="https://user-images.githubusercontent.com/541093/126355368-04f48e69-4ba1-45f4-bc2f-85c7f2805814.png">
<img width="932" alt="Screen Shot 2021-07-20 at 11 49 36 AM" src="https://user-images.githubusercontent.com/541093/126355410-e1254d18-45b3-4e89-b635-3078bb328610.png">

### How to test the changes in this Pull Request:

1. Apply the patch
2. View source, check that the meta tags are output
3. Check that they're not broken and make sense
